detox/test/scripts/postinstall.js

```diff
@@ -1,29 +1,66 @@
+const fs = require('fs-extra');
+const path = require('path');
+const semver = require('semver');
+
 const rnVersion = function() {
   const rnPackageJson = require('react-native/package.json');
   return rnPackageJson.version;
 }();
 
 function patchHermesLocationForRN60Android() {
-  const semver = require('semver');
-  const fs = require('fs-extra');
-  const path = require('path');
+  const HERMES_PATH_ROOT = path.join('node_modules', 'hermesvm');
+  const HERMES_PATH_RN = path.join('node_modules', 'react-native', 'node_modules', 'hermesvm');
```

Comment:
You could use `path.join('node_modules', 'react-native/node_modules/hermesvm')` — Node.js will fix Win/POSIX slash direction automatically... or `path.resolve(...)` maybe could work out here as well... 🤔